src/components/themes/blueradiance/theme.css

```diff
@@ -451,7 +446,6 @@ html {
 }
 
 ::-webkit-scrollbar-track {
-    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, .3)
 }
```

Comment:
Looks like this rule can be removed.

Reply:
I ended up leaving these in, but non-prefixed (since autoprefixer takes care of that if needed).

It still works in webkit but, for some reason, autoprefixed didn't re-add the rule.